src/synchronization/follower-orchestration.ts

```
@@ -692,23 +739,44 @@ const removedResourceState = (
           symlinkTo: applied.symlinkTo,
         },
       };
     case "directory":
     case "skill":
       if (applied.ownedFiles === undefined) return undefined;
+      const directories = applied.ownedFiles
+        .filter((file) => file.objectKind === "directory")
+        .map((directory) => ({
+          path: directory.path,
+          mode: directory.mode ?? 0o700,
+        }));
```

Biome (2.5.8)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)